README.md

```diff
@@ -125,12 +119,9 @@ emulator-5554 unauthorized
 
 ![-----------------------------------------------------](https://raw.githubusercontent.com/andreasbm/readme/master/assets/lines/aqua.png)
 
-## Roadmap
+## Roadmap and ToDo
 
 See the [open issues](https://github.com/yuvrajraghuvanshis/WhatsApp-Key-Database-Extractor/issues) for a list of proposed features (and known issues).
-#### ToDo
-* ![Status](https://img.shields.io/badge/Status-Done-success) Zip extracted folder with password.
-* ![Status](https://img.shields.io/badge/Status-Removed-red) Command line arguments
```

Comment:
Why'd you do that.

Reply:
They're not really todo als they're both done. And I thought closed ToDos are more or less under issues anyway.

Reply:
Yes, here they are also serving purpose of history and roadmap of project. One does not need to search through issue for them.